solution.txt

```diff
@@ -0,0 +1,51 @@
+// "Get Even"
+for (let i = 0; i <= 200; i+=2){
+  console.log(i);
+}
+
+// "Excited Kitten Part 1"
+for ( let i = 0; i < 10; i++){
+  console.log("Love me, pet me! HSSSSSS!");
+}
+
+// " Excitd Kitten Part 2"
+for ( let i = 0; i < 10; i++){
+  const sentences = ["...human...why you taking pictures of me?...", "...the catnip made me do it...", "meow?", "...why does the red dot always get away..."];
+  if (i % 2 === 0) {
+    const randomNumber = Math.floor(Math.random()*sentences.length);
+    console.log(sentences[randomNumber])
+  }
+}
```

Comment:
nice start here! if you like we can talk through this, it's definitely a lot to think through!